packages/react-native-navigation/src/__tests__/rum/instrumentation/DdRumReactNativeNavigationTracking.test.tsx

```diff
@@ -120,6 +120,29 @@ it('M send a RUM ViewEvent W startTracking() componentDidAppear', async () => {
 
 
 
+it('M send a RUM ViewEvent W startTracking() componentDidAppear { custom viewPredicate }', async () => {
+    console.log("!!!!!!!")
+    // GIVEN
+    let componentId = "component42"
+    const customViewName = "custom_view_name"
+    const predicate: ViewNamePredicate = function customViewNamePredicate(_trackedView: any, _trackedName: string)  { 
```

Comment:
minor: if function name is not used anonymous lambda can be written instead, it will be shorter to write